todo-app/src/app/Services/todo-data.service.ts

```diff
@@ -0,0 +1,50 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Todo } from '../Models/todo';
+import {Observable} from "rxjs";
+import { map } from "rxjs/operators";
+import { IResponse } from "../Models/res.model";
+
+@Injectable({
+  providedIn: 'root'
+})
+
+export class TodoDataService {
+  baseUrl = 'https://todolist-89791.firebaseio.com/todos';
+
+  constructor(
+    private http: HttpClient
+  ) { }
+
+  listTodo() {
+    return this.http
+      .get<Todo[]>(`${this.baseUrl}.json`)
+      .pipe(map( todos => {
```

Comment:
Try to rewrite this pipe to the conditional operator

Reply:
need help...

Reply:
instead of 'if' we could write with ternary operators. For example (item) ? 'yes' : 'no'